wordpress-nginx/roles/wordpress/tasks/main.yml

```diff
@@ -13,10 +16,12 @@
   user: name=wordpress group=wordpress home=/srv/wordpress/
 
 - name: Fetch random salts for WordPress config
-  local_action: command curl https://api.wordpress.org/secret-key/1.1/salt/
+  get_url:
```

Comment:
This should remain local

Reply:
`ansible-lint` error here was
```
[303] curl used in place of get_url or uri module
wordpress-nginx/roles/wordpress/tasks/main.yml:15
Task/Handler: Fetch random salts for WordPress config
```

Reply:
Exactly:
> [504] Do not use 'local_action', **use 'delegate_to: localhost'**

Reply:
https://github.com/ansible/ansible-examples/pull/278/commits/fe001733534b7f6bf2d418c627c5bbb091d9584f